packages/components/src/components/static-url/static-url.jsx

```diff
@@ -1,11 +1,16 @@
 import React from 'react';
-import { getStaticUrl, PlatformContext } from '@deriv/shared';
+import { getStaticUrl, PlatformContext, setUrlLanguage } from '@deriv/shared';
+import { getLanguage } from '@deriv/translations';
 
 const StaticUrl = ({ href, is_document, children, ...props }) => {
     const { is_appstore } = React.useContext(PlatformContext);
+    const getHref = () => {
+        setUrlLanguage(getLanguage());
```

Comment:
This looks like a temporary solution, why can't we set the `default_language` from `app.jsx` after selecting a different language?

Reply:
I'm afraid it will not help, as our current implementation of getStaticUrl work synchronously and can take language variable only from the same file. (outer scope for it). But  changing the language is going asynchronously.
others possible options for solving this issue:
1) get the language option from common store and pass it like additional parameter to getStaticUrl function. In this case we should add connect to GoToDeriv components and pass language props to StaticUrl component.
2) call setUrlLanguage(getLanguage()) inside getStaticUrl, in this case local variable will be defined with proper value.
but in this case we should check different places of usage of this function, if this changes doesn’t affect any other flows